src/styles/card.scss

```diff
@@ -0,0 +1,155 @@
+@import './variables';
+@import './mixins';
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @include on-mobile {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @include on-tablet {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @include on-desktop {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  width: 198px;
+  height: 404px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: all 0.3s ease-in-out;
+
+  &__photo {
+    display: flex;
+    justify-content: center;
+    margin: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: $big-line-height;
+    margin: 40px 16px 0;
+
+    & > p {
+      margin: 0 0 4px;
+    }
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__subtitle {
+    color: $secondary-accent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: $default-line-height;
+    margin: 0 16px;
+    & > p {
+      margin: 0;
+    }
+  }
+
+  &__feedback {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin: 16px 16px 24px;
+
+    &-review {
+      display: flex;
+      color: $main-accent;
+      font-size: 10px;
+      font-weight: 400;
+
+      & > p {
```

Comment:
Also, do not use html tags as css selectors, if needed add a new class [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Don%27t%20use%20tag%20names%20for%20styling%20(except%20html%20and%20body))

Reply:
I spent the whole day checking those and I still left some behind.... LOL